app/controllers/map_controller.rb

```diff
@@ -2,8 +2,14 @@ class MapController < ApplicationController
   skip_before_action :authenticate_user!
 
   def index
-    schools = School.includes(:configuration).visible.order(:name)
-    schools = schools.where(school_group_id: params[:school_group_id]).order(:name) if params[:school_group_id]
+    schools = School.visible
```

Comment:
If the schools controller is showing different schools for admin users (as in not visible), should the map too?